_layouts/sidebar.html

```diff
@@ -24,6 +24,9 @@ <h4>Examples</h4>
               <li>
                 <a href="/v1/examples/syncing-employees">Syncing Employees</a>
               </li>
+              <li>
+                <a href="/v1/examples/creating-a-company">Creating a Company</a>
```

Comment:
I ended up going with an example because this section of the docs is quite long and got lost on the companies page. Also, the request/response attributes don't line up with the attributes for a company which could be confusing.